tests/pulse/test_transmon.py

```diff
@@ -181,3 +465,146 @@ def test_add_two_settings_with_one_anharmonicity_None(
 
         settings10 = settings1 + settings0
         assert settings10.anharmonicity == [0.0] * len(omega0) + anharmonicity
+
+
+class TestIntegration:
+    @pytest.mark.jax
+    def test_jitted_qnode(
+        self,
+    ):
+        """Test that regular and jitted qnode yield same result"""
+        import jax
+        import jax.numpy as jnp
+
+        Hd = transmon_interaction(omega, connections, g, wires=wires)
+
+        def fa(p, t):
+            return jnp.polyval(p, t)
+
+        def fb(p, t):
+            return p[0] * jnp.sin(p[1] * t)
+
+        Ht = transmon_drive(amplitude=fa, phase=fb, freq=0.5, wires=[0])
+        H = Hd + Ht
+
+        dev = qml.device("default.qubit.jax", wires=wires)
+
+        ts = jnp.array([0.0, 3.0])
+        H_obj = sum(qml.PauliZ(i) for i in range(2))
+
+        @qml.qnode(dev, interface="jax")
+        def qnode(params):
+            qml.evolve(H)(params, ts)
+            return qml.expval(H_obj)
+
+        @jax.jit
+        @qml.qnode(dev, interface="jax")
+        def qnode_jit(params):
+            qml.evolve(H)(params, ts)
+            return qml.expval(H_obj)
+
+        params = (jnp.ones(5), jnp.array([1.0, jnp.pi]))
+
+        res = qnode(params)
+        res_jit = qnode_jit(params)
+
+        assert isinstance(res, jax.Array)
+        assert qml.math.isclose(res, res_jit)
+
+    @pytest.mark.jax
+    def test_jitted_qnode_multidrive(
+        self,
+    ):
```

Comment:
```suggestion
    def test_jitted_qnode_multidrive(self):
```
Terrible update they made to black :D

Reply:
if I accept this change and run black locally again this will be again reformatted 😅

Reply:
No, it shouldn't: The trailing comma after `self` makes the difference!

Reply:
If it actually does, then excuse my outdated suggestions :sweat_smile: